client/client-multi/client-cpp/impl/channel/impl/DefaultOperationTcpChannel.cpp

```diff
@@ -252,6 +252,10 @@ void DefaultOperationTcpChannel::openConnection()
         return;
     }
 
+    /* Check whether ip has been changed */
```

Comment:
Strange comment. As I understand from the line below, you don't check but noify about a local address.